### ISSUE TITLE
Scheduler Component

### ISSUE THREAD

**Issue by WillianGruber** [Megabit/Blazorise#475]
It would be great to have a Scheduler Component... I know that it must be hard to implement one, but maybe it could be done...

**Comment by stsrki** [comment]
I agree but for now I don't have plan to do to it. All my work is going to be for v0.9 from now on, and after that I will consider a Scheduler Component.

**Comment by WillianGruber** [comment]
For reference:
https://openui5.hana.ondemand.com/entity/sap.m.PlanningCalendar

**Comment by stsrki** [comment]
https://www.reddit.com/r/dotnet/comments/1gfwrlh/creating_a_monthly_schedule_best_approach_for/?share_id=YCQ7izossv4A0x8JH23Q3&utm_content=1&utm_medium=android_app&utm_name=androidcss&utm_source=share&utm_term=2
